### PR TITLE
SLOW 

### PR DESCRIPTION
**Pull request by mewzax**

### CODE REVIEW

**Comment by slowwdev** [comment]
PLS

File: domains/slow.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "slow personal developer website",
+    "repo": "https://github.com/slowwdev/slow",
```
Comment (TheBotlyNoob) [inline domains/slow.json:3]: you probably want to rename this repo to `slowdev.github.io`

File: domains/slow.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "slow personal developer website",
+    "repo": "https://github.com/slowwdev/slow",
+    "owner": {
+      "username": "slowwdev",
+      "email": "slowdev14@gmail.com"
+    },
+    "record": {
+      "CNAME": "slow.github.io"
```
Comment (TheBotlyNoob) [inline domains/slow.json:9]: and change this to `slowdev.github.io`

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Comment by slowwdev** [comment]
i changed

**Comment by slowwdev** [comment]
@TheBotlyNoob

**Review by phenax — APPROVED** [review]

**Pull request merged**